src/Microsoft.VisualStudio.Editor.Razor/RazorDirectiveCompletionProvider.cs

```diff
@@ -111,13 +103,6 @@ public override Task ProvideCompletionsAsync(CompletionContext context)
                 return Task.CompletedTask;
             }
 
-            if (!_textBufferProvider.TryGetFromDocument(context.Document, out var textBuffer) ||
-                !_asyncCompletionBroker.IsCompletionSupported(textBuffer.ContentType))
```

Comment:
We shouldn't be the one loading extra IAsyncCompletion bits if we're not operating on a Razor file.